check_vm_backup.ps1

```diff
@@ -18,17 +19,32 @@ if($vm -eq $FALSE) {
     exit 3
 }
 
-$now=[Math]::Floor([decimal](Get-Date(Get-Date).ToUniversalTime()-uformat "%s"))
+if($vm -eq $FALSE) {
+    echo "param -type is missing and must either be Backup or Copy. Leave it empty for Backupjobs"
```

Comment:
Why did you replace " echo "param -vm missing" with this?

Reply:
Variable $vm not changed to $type after copying it. I corrected this failure. param -vm missing is still available.